edx-platform/pearson-kaust-theme/lms/static/sass/partials/lms/theme/_support.scss

```diff
@@ -0,0 +1,20 @@
+// Styles for the support modal.
+
+.panel-heading {
+  background: #6a6d6d !important;
+  border: #6a6d6d !important;
```

Comment:
@Squirrel18 this may be a variable like $primary?

Reply:
Good catch @luismorenolopera Addressed.